src/services/account.ts

```diff
@@ -143,6 +144,7 @@ export class Account {
     /**
      * Create new billing address
      *
+     * Add a new billing address to a user&#039;s account.
```

Comment:
Weird encoding here, does it cause issues? Is it already present elsewhere?

Reply:
Doesn't seem to cause issues, VSCode appears to automatically convert it back into normal and this already exists in other parts of the SDK. See: 
https://github.com/appwrite/sdk-for-console/blob/f55c80f8b4750ce5ff25b8aa0c1b28b0c0ded3d2/src/services/storage.ts#L283-L294

<img width="965" alt="image" src="https://github.com/user-attachments/assets/8b775ca1-9121-4e03-b703-65f3ef10d771" />